src/lib/multipageloader.cc

```diff
@@ -107,6 +107,20 @@ QNetworkReply * MyNetworkAccessManager::createRequest(Operation op, const QNetwo
 	return QNetworkAccessManager::createRequest(op, r3, outgoingData);
 }
 
+MyNetworkProxyFactory::MyNetworkProxyFactory (QNetworkProxy proxy, QList<QString> bph, ResourceObject & res):
```

Comment:
Why do you need the `ResourceObject` here? It isn't used, so should ideally be removed.